pkg/labels/labels.go

```diff
@@ -7,4 +7,6 @@ const (
 	MasterRole = "node-role.kubernetes.io/master"
 	// ControlPlaneRole is the new role label of control plane nodes
 	ControlPlaneRole = "node-role.kubernetes.io/control-plane"
+	// DefaultTemplate is a label that would be set in case a remediator may have several remediation templates in order to signal to the UI which one is the default
```

Comment:
Good idea to have the label in common.
I'd like to come up with a shorter description. What do you think about something like the following?

DefaultTemplate label indicates to third party tools (e.g. UI) the default remediation template in case several exists.